packaging/amazon-linux-ami/ipCleanup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
I would recommend doing probably `set -ex` at the top of these scripts. -e tells the scripts to fail if any of the commands below fail, -x prints out what the script does so we can see exactly what it's doing in the journalctl logs.